private/bufpkg/bufplugin/bufpluginconfig/testdata/success/go-empty-registry/buf.plugin.yaml

```diff
@@ -0,0 +1,8 @@
+version: v1
+name: buf.build/library/go
+plugin_version: v1.28.1
+default_opts:
+  - paths=source_relative
+output_languages: [go]
+registry:
+  go: {}
```

Comment:
Example of enabling a plugin for use by Go remote registry without requiring additional fields to be specified.